### DISCUSSION TITLE
Regarding fallback to cloudwatch if writing logs to kafka topics fails Quarkus-2.16.9 version

### DISCUSSION THREAD

**Discussion by NeetuMalik** [quarkusio/quarkus#35452] [Q&A]
Hello,
I'm currently working on AWS Lambda and a task manager built using Quarkus version 2.16.9 with Java. One of our requirements involves disabling log writing to CloudWatch, as we're already sending logs to Kibana. Our approach for logging to Kibana involves sending logs to a Kafka topic and then forwarding them to Kibana via a Logstash Elastic cluster.

To disable logs from being written to CloudWatch, I've made the following changes in the application.properties file. Initially, it was logging to the console and then to CloudWatch:

properties
 -Disable console logging
quarkus.log.console.enable=false

-Disable CloudWatch logging
quarkus.log.handler.console."console-handlers".enable=false
However, I'm currently searching for properties or mechanisms that could be employed in case writing logs to the Kafka topic fails. In such a scenario, I'd like to have a fallback option to log to CloudWatch. Below are the properties in the application.properties file that are used for logging to Kafka:

properties
 -Enable Kafka logging
quarkus.log.handler.kafka.enabled=true
quarkus.log.handler.kafka.broker-list=localhost:29092
quarkus.log.handler.kafka.topic=test-logs
quarkus.log.handler.kafka.level=INFO
quarkus.log.category."io.quarkus".level=INFO
quarkus.log.handler.kafka.async=true
I'm open to any suggestions or guidance on implementing this fallback mechanism.

[a comment by an automated account is not shown]

**Reply by cescoffier** [reply]
I don't think we have a way to do that if you use the log appender directly.
